srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -38,6 +41,25 @@
     public static final String VISTA_DATE_OUTPUT_FORMAT = "MM/dd/yy@HH:mm";
     
     private static final Map<String, String> TRANSLATION_MAP;
+    
+    /**
+     * Using a map instead of a different type of collection for efficiency when
+     * filtering for health factors.
+     */
+    private static final Map<String, Boolean> VALID_HEALTH_FACTORS;
+    
+    private static final String[] HEALTH_FACTORS_ARRAY = { "ALCOHOL - TREATMENT REFERRAL", "ALCOHOL USE",
```

Comment:
This constant could use a comment, and I'm not sure if this is the best place for it. It's not RPC-specific. How about `VistaPatientDao` itself?